openpype/scripts/otio_burnin.py

```diff
@@ -474,7 +474,9 @@ def render(self, output, args=None, overwrite=False, **kwargs):
                                "use overwrite" % output)
 
         is_sequence = "%" in output
+
         duration = kwargs.get("duration", 0)
+        frame_id = kwargs.get("frame_id", 0)
```

Comment:
`duration` isn't used anymore and can be remove

Reply:
indeed